src/main/java/net/pms/store/MediaScanner.java

```diff
@@ -504,19 +525,24 @@ private static void removeFileEntry(String filename) {
 			 * give us information about those new files, as it wasn't listening
 			 * when they were created, so make sure we parse them.
 			 */
+			File f = new File(filename);
 			if (isDir) {
 				if (ENTRY_CREATE.equals(event)) {
-					addFolderEntry(new File(filename));
+					addFolderEntry(f);
 				} else if (ENTRY_DELETE.equals(event)) {
 					removeFolderEntry(filename);
 				}
 			} else {
 				if (ENTRY_CREATE.equals(event)) {
-					parseFileEntry(new File(filename), true, false);
+					parseFileEntry(f, true, false);
 				} else if (ENTRY_DELETE.equals(event)) {
 					removeFileEntry(filename);
+				} else if (ENTRY_MODIFY.equals(event)) {
+					parseFileEntry(f, false, false);
```

Comment:
@SubJunk Is Line 541 still needed? Not sure, what's exactly suppose to do.

Reply:
Looks like it re-parses a modified file, but I'm not sure if it works or just hits the cache